### PR TITLE
delete tag modal

### PR DESCRIPTION
**Pull request by ahsu1230**
@andrewconner @yipingliao 
looks very similar to the delete library modal popup

### CODE REVIEW

**Comment by andrewconner** [comment]
Thanks

File: kifi-backend/modules/shoebox/angular/src/tagManage/deleteTag.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      restrict: 'A',
+      require: '^kfModal',
+      templateUrl: 'tagManage/deleteTagMsg.tpl.html',
+      link: function (scope, element, attrs, kfModalCtrl) {
+
+        scope.deleteTag = function () {
+          tagService.remove(scope.modalData.tag);
+          _.remove(scope.modalData.tagsToShow, function(t) { return t === scope.modalData.tag; });
+          kfModalCtrl.close();
+          return scope.modalData.tagsToShow;
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/tagManage/deleteTag.js:17]: Why does this need its own directive? And who is looking at this returned data?

File: kifi-backend/modules/shoebox/angular/src/tagManage/deleteTag.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+angular.module('kifi')
+
+.directive('kfDeleteTag', ['$location', 'tagService',
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/tagManage/deleteTag.js:5]: `$location` is not being used.

File: kifi-backend/modules/shoebox/angular/src/tagManage/deleteTagMsg.tpl.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<div kf-modal class="kf-delete-tag-confirm-modal">
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/tagManage/deleteTagMsg.tpl.html:1]: You have two `kf-modal`s nested now.

**Pull request merged**